### PR TITLE
more cargo cart fixes

### PR DESCRIPTION
**Pull request by gurfan**
Fixed issue with wrenching things on the cart for useable-unanchored machines, made blacklisted items anchored but unpowered

### CODE REVIEW

File: code/game/objects/structures/vehicles/carts/cargo.dm [hunk 1]
```diff
@@ -19,10 +20,14 @@
...
 
-/obj/machinery/cart/cargo/process()			//This might cause lag....?
-	if(internal_battery.charge == 0 && loaded_machine)
-		loaded_machine.power_change()
+/obj/machinery/cart/cargo/process()	
+	if(internal_battery)
+		if(internal_battery.charge == 0 && loaded_machine)
+			loaded_machine.power_change()
 
+/obj/machinery/loaded_machine/wrenchAnchor(mob/user, obj/item/I)
```
Comment (jknpj) [inline code/game/objects/structures/vehicles/carts/cargo.dm]: :thonk:

Comment (jknpj) [reply code/game/objects/structures/vehicles/carts/cargo.dm]: weird place to leave this

Comment (gurfan) [reply code/game/objects/structures/vehicles/carts/cargo.dm]: forgot to remove that, oops

File: code/game/objects/structures/vehicles/carts/cargo.dm [hunk 2]
```diff
@@ -116,16 +117,18 @@
 	
 	if(istype(C, /obj/machinery))
 		loaded_machine = C
+		loaded_machine.anchored = 1
+		loaded_machine.battery_dependent = 1
+		if(loaded_machine.machine_flags & WRENCHMOVE)
```
Comment (DamianX) [inline code/game/objects/structures/vehicles/carts/cargo.dm]: this check is not necessary

File: code/game/objects/structures/vehicles/carts/cargo.dm [hunk 3]
```diff
@@ -138,16 +141,18 @@
...
-			loaded_machine.battery_dependent = 1
-			loaded_machine.power_change()
-			visible_message("The [load]'s cables disconnect from the cart.'")
-		loaded_machine = null
+		loaded_machine.state = 0
+		loaded_machine.anchored = 0
+		loaded_machine.battery_dependent = 0
+		loaded_machine.connected_cell = null
+		loaded_machine.power_change()
+		if(!(loaded_machine.machine_flags & WRENCHMOVE))
```
Comment (DamianX) [inline code/game/objects/structures/vehicles/carts/cargo.dm]: this check is not necessary

**Pull request merged**